tests/setup-tests.js

```diff
@@ -11,6 +11,9 @@ require("../ace/mode-python_microbit");
 require("../ace/snippets/python_microbit.js");
 require('../ace/ext-language_tools.js');
 
+//fs lib for file check tests
+global.fs = require("fs");
```

Comment:
Rather than add these globally, it's better to require them in the spec files that need it.

Reply:
Moved to spec file